docs/drafts/downgrading-an-extension.md

```diff
@@ -0,0 +1,211 @@
+
+# Downgrade a ClusterExtension
+
+## Introduction
+
+Downgrading a `ClusterExtension` involves reverting the extension to a previous available version. This process might be required due to compatibility issues, unexpected behavior in the newer version, or specific feature requirements that are only available in an earlier release. This guide provides step-by-step instructions to safely downgrade a `ClusterExtension`, including necessary overrides to bypass default constraints and disable CRD safety checks.
```

Comment:
I have a bit of an aversion to saying that this guide will help you downgrade "safely". IMO, performing a downgrade with the type of software we are managing with OLM is inherently an "unsafe" operation. While it may not always be the case, my assumption would be that a user runs the risk of data loss, invalidation of anything stored at new CRD versions, and potentially break clients that expect the newer version to be running. Before doing a downgrade, users should be confident that they want to do it and that they understand and have considered the potential consequences of performing a downgrade.

Reply:
Yeah, that's fair. I have reworded the intro entirely, incorporating your feedback. Thanks!